### ISSUE TITLE
List of authors is broken

### ISSUE THREAD

**Issue by peacekeeper** [w3c/did-spec#74]
The spec right now doesn't have the full list of authors it had before; I think this broke in 0c4189867b49ab7fda3034599dd4fa6e356e4736 and cda1698e7685dd82f8547cac16d6c15bf68161be. Should we restore the original list?

**Comment by msporny** [comment]
> Should we restore the original list?

Yes, we should at least restore the original list (in the order it appears).

We should also discuss the original list, as it's arguable that others should be on that list that have contributed more than the folks on the original list. We always have that discussion toward the end of CR, but it may not hurt to have that discussion before then. That is a separate discussion though... for now, we should replace the original list of authors.

**Comment by burnburn** [comment]
>> Should we restore the original list?

> Yes, we should at least restore the original list (in the order the authors appeared before).

Agreed.

**Comment by brentzundel** [comment]
fixed in #78
